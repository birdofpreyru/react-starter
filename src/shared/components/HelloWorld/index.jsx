/**
 * A simple Hello World page.
 */

import React from 'react';

import './style.scss';

export default function HelloWorld() {
  return (
    <div>
      <h1 styleName="title">
        Hello World!
      </h1>
      <p>
        Dr. Pogodin&apos;s React Starter is up!
      </p>
    </div>
  );
}
