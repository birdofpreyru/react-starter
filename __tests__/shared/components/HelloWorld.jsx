import HelloWorld from 'components/HelloWorld';
import React from 'react';
import { JU } from 'topcoder-react-utils';

test('Matches shallow snapshot', () => {
  JU.shallowSnapshot(<HelloWorld />);
});
