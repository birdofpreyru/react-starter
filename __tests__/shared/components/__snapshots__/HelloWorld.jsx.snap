// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches shallow snapshot 1`] = `
<div>
  <h1
    className="src-shared-components-HelloWorld-___style__title___UXPoT"
    data-tip="Tooltip Example"
  >
    Hello World!
  </h1>
  <p>
    Dr. Pogodin's React Starter is up!
  </p>
</div>
`;
